projects/clas12/targets/materials.py

```diff
@@ -178,7 +163,6 @@ def define_materials(configuration):
 	gmaterial.publish(configuration)
 
 	# ND3, not sure if G4 has H2 material so used H for now
-	# @mariakzurek: What to do with the comment above? This target is using G4_H
```

Comment:
What should I do with the `ND3` material defined here with the same name as the material defined in `g4world.cc`? It uses H instead of D. The `solidND3` material uses `ND3`, but I'm not sure if it uses the one defined in g4world.cc or the one defined here.

Reply:
This can be removed from materials.py and 'ND3' (predefined in g4world) can be used.